sample.env

```diff
@@ -373,3 +373,6 @@ CUSTOM_PARAMETERS=[["userdata-bbb_auto_swap_layout","true"],["userdata-bbb_show_
 # streaming parameter
 HIDE_CHAT=true
 HIDE_USER_LIST=true
+
+# mp4 url
```

Comment:
MP4 URL - The domain of the BIGBLUEBUTTON_ENDPOINT parameter. It could either be a standalone BigBlueButton server or a Scalelite server. Please install https://github.com/manishkatyan/bbb-mp4 on each of the BigBlueButton servers to automatically convert recordings into MP4.